tests/src/runtimeApi/memory/hipMemcpy.cpp

```diff
@@ -34,6 +34,12 @@ THE SOFTWARE.
 #include "hip/hip_runtime.h"
 #include "test_common.h"
 
+#ifdef _WIN32
+#include <windows.h>
```

Comment:
could you please add 
#define WIN32_LEAN_AND_MEAN
before #include <windows.h>
this is to avoid linking extra libs which we do not need.